src/ConactApp/widget/ConactApp.js

```diff
@@ -78,57 +81,55 @@ define([
             //this.inputText.value = "test";
             // this.selectTo.value="test";    
             var preLoadValue = [];
-            var preLoadFromValue = this._contextObj.jsonData.attributes.From.value;
-            var preLoadToValue = this._contextObj.jsonData.attributes.To.value;
+            var preLoadFromValue = this._contextObj.jsonData.attributes.From.value; //use get("attribute")
+            var preLoadToValue = this._contextObj.jsonData.attributes.To.value; //unused variable?
             if (this.maxItems === 1) {
-                preLoadValue.push(this._contextObj.jsonData.attributes.From.value);
+                preLoadValue.push(this._contextObj.jsonData.attributes.From.value); //? From ?
             } else {
-                var toValue = this._contextObj.jsonData.attributes.To.value;
-                if (toValue.indexOf(',') > -1) {
+                var toValue = this._contextObj.jsonData.attributes.To.value; //where "To" come from?
+                if (toValue.indexOf(',') > -1) { //do we need this?
```

Comment:
Yep, the value returns here in the form of a string of emails separated by comma (,)